From 0dac9ad5773c1b220029143de5619785ddb09450 Mon Sep 17 00:00:00 2001
From: kopal <kopal@broadcom.com>
Date: Thu, 16 Jan 2014 12:14:32 +0530
Subject: [PATCH 0476/1532] XLP2xx_OOB: Automatically switch_root to the Hard
 Disk.

Added root=auto as a Kernel commandline parameter so that the Kernel mounts
and switch_root to a hard disk.
[Based on SDK 3.2]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/mips/boot/dts/xlp2xx-ep-linux.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/boot/dts/xlp2xx-ep-linux.dts b/arch/mips/boot/dts/xlp2xx-ep-linux.dts
index c95fcb1..ef41db1 100644
--- a/arch/mips/boot/dts/xlp2xx-ep-linux.dts
+++ b/arch/mips/boot/dts/xlp2xx-ep-linux.dts
@@ -78,7 +78,7 @@
 	};
 
 	chosen {
-		bootargs = " console=ttyS0,115200 rhash_entries=1048576 ";
+		bootargs = " root=auto console=ttyS0,115200 rhash_entries=1048576 ";
 		/*linux,initrd-start=<0x00c00000>;
 		linux,initrd-end=<0x02c00000>;*/
 	};
-- 
1.9.1

